src/Elements/Core/References.ts

```diff
@@ -0,0 +1,55 @@
+import { AbstractNode } from "../../Abstracts/AbstractNode";
+import { AttributeGroupItem } from "../../Interfaces/AttributeGroupItem";
+import { NodeRules } from "../../Interfaces/NodeRules";
+import { Source } from "../../AttributeGroups/Source";
+
+/**
+ * The element References is a metadata container of all the references
+ * to entities external to the document mentioned in the document.
+ * They include references to legal documents of any form,as well as
+ * references to people, organizations, events, roles, concepts, and
+ * anything else is managed by the Akoma Ntoso ontology.
+ */
+export class References extends AbstractNode {
```

Comment:
Element of type `RefItems`

Reply:
Is this a rename suggestion? I didn't find this already existing.

Reply:
the `CHILDREN_MAP`, `SEQUENCE` and `ATTRIBUTE_GROUPS` should extend `refItems`